config/config.example.json

```diff
@@ -21,5 +21,6 @@
       }
     }
   },
-  "token": "TOKEN"
+  "token": "TOKEN",
+  "org": "localdev"
```

Comment:
I don't think we have a deco config file here, but we'll need to add this to the Jenkins jobs

Reply:
yeah I looked for it and then remember we just use jenkins for this one